source/quick-start.txt

```diff
@@ -27,6 +27,13 @@ MongoDB Atlas is a fully-managed cloud database service that hosts your data
 on MongoDB clusters. In this guide, we show you how to get started with your
 own free (no credit card required) cluster.
 
+.. note::
```

Comment:
@rustagir thoughts on this as a note? I originally had it as plain text in the `Next Steps` section but it seemed like it would get overlooked if it was down there.

Reply:
I think it is fine as an admonition, but maybe we can make it a TIP box instead!